docs/running-on-kubernetes.md

```diff
@@ -768,6 +768,53 @@ from the other deployment modes. See the [configuration page](configuration.html
     <code>myIdentifier</code>. Multiple node selector keys can be added by setting multiple configurations with this prefix.
   </td>
 </tr>
+<tr>
+  <td><code>spark.kubernetes.kerberos.enabled</code></td> 
+  <td>false</td>
+  <td>
+    Specify whether your job is a job that will require a Kerberos Authorization to access HDFS. By default, we
```

Comment:
s/Specify whether your job is a job that will require a Kerberos Authorization to access HDFS/Specify whether your job requires a Kerberos Authentication to access HDFS/